global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "8.0.100"
+    "version": "8.0.202"
```

Comment:
Nitpick: you should not bump the SDK here, it is unrelated to what you are trying to fix.
On the flip-side I'm not against having this, make it 8.0.200 and `"rollForward": "latestMinor"`

Reply:
I saw that but I do not know how to force it down. The 8.0.202 sdk is what is installed on my dev environment and I had to change it to work. For future reference, should the intended changes be put into into a new (local) branch and then PR'd to make your (maintainers) job easier?

Reply:
> I saw that but I do not know how to force it down

I told you how to, add `"rollForward": "latestMinor"` and `"version": "8.0.200"` under `"sdk`.

Just because you installed `.202` does not mean every other contributor needs that exact version. `.200` and up should be fine.